### PR TITLE
Parallel tests

### PR DESCRIPTION
**Pull request by lu-zero**

### CODE REVIEW

**Comment by lu-zero** [comment]
Ok, this way seems to make travis happy, I'd squash and merge.

File: src/test_encode_decode.rs [hunk 1]
```diff
@@ -105,22 +105,47 @@ fn setup_encoder(
...
+            #[test]
+            fn [<speed_ $S>]() {
+                speed($S)
+            }
+        }
+    )*
+  }
+}
+
+test_speeds!{ 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10}
```
Comment (barrbrain) [inline src/test_encode_decode.rs]: Whitespace is a little odd here.
```suggestion
test_speeds!{ 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10 }
```

**Comment by tdaede** [comment]
squash please

File: src/test_encode_decode.rs (hunk 1 above)
Comment (lu-zero) [reply src/test_encode_decode.rs]: Folded in the rework.

**Comment by lu-zero** [comment]
I'd push it today if nobody is against it.

**Comment by tdaede** [comment]
Please do

**Pull request merged**